graphql/subcourse/fields.ts

```diff
@@ -86,6 +87,12 @@ export class ExtendedFieldsSubcourseResolver {
                     { joinAfterStart: { equals: true }, lecture: { some: { start: { gt: new Date() } } } },
                 ],
             });
+            if (isSessionPupil(context)) {
+                const pupil = await getSessionPupil(context);
+                filters.push({
+                    AND: [{ minGrade: { lte: gradeAsInt(pupil.grade) }, maxGrade: { gte: gradeAsInt(pupil.grade) } }],
```

Comment:
What if pupil.grade is null?

Reply:
I think than the pupil shouldn't get a suggestion or? @Jonasdoubleyou